### ISSUE TITLE
"fix": Exceptions are not returned to the client

### ISSUE THREAD

**Issue by GrigoriiBerezin** [EpamLifeSciencesTeam/random_coffee#9]
**Problem:**
We have cases, when errors are happened, but the client will se only that something goes wrong without any error information to avoid it in future and simply understand what's going on. Let's think about error handler to let clients know about the mistakes that they were made.

**Solution:**
We must add error handler to make returning errors info possible. ([ExceptionHandler docs](https://doc.akka.io/docs/akka-http/current/routing-dsl/exception-handling.html))
I don't, it should be discussed with @myazinn, but I think, we should have exception handler for each service.

**Steps to reproduce:**

1. Create user by using signUp endpoint (localhost:8088/auth/v1/signUp)
2. Try to create another user using the same email

**Actual result:**
Response with 500 status and "Internal server error" message

**Expected result:**
Json response with 500 status and body like
`{
   "message": "error message"
}`

**Comment by DruzhininPavel** [comment]
I think it should be the 409 error "Resource already exist" or something like this, because 500 it is Internal server error, it happense when server is down or when it is the critical error.

**Comment by GrigoriiBerezin** [comment]
Yep, probably. We should define error ADT for every server and than mapping them to statuses as we did on [cromwell-pipeline](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/blob/develop/controllers/src/main/scala/cromwell/pipeline/controller/ProjectConfigurationController.scala)
